gsa/src/gmp/commands/tlscertificates.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import registerCommand from '../command';
+
+import TlsCertificate from '../models/tlscertificate';
+
+import EntitiesCommand from './entities';
+import EntityCommand from './entity';
+
+class TlsCertificateCommand extends EntityCommand {
+  constructor(http) {
+    super(http, 'tls_certificate', TlsCertificate);
+  }
+
+  getElementFromRoot(root) {
+    return root.get_tls_certificate.get_tls_certificate_response.asset;
+  }
+}
+
+class TlsCertificatesCommand extends EntitiesCommand {
+  constructor(http) {
+    super(http, 'tls_certificate', TlsCertificate);
+  }
+
+  get(params = {}, options) {
+    return super.get({...params, details: 1}, options);
```

Comment:
Is this really necessary? Do we need to send details=1?

Reply:
I included it for the first implementation in order to test things and because I didn't know what to put into the details. As of now, this is not really necessary, but to be perfectly honest, I don't want to spent a lot of time changing these things for the PR and then immediately change them back to continue developing. For the final implementation this might look different ;)

Reply:
Just remove the get method completely and use the one from the parent class :)

Reply:
Done :)